src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -653,5 +653,10 @@ public override IFileSystemObject GetFileSystemInfo(string fullPath, bool asDire
                 (IFileSystemObject)new DirectoryInfo(fullPath, null) :
                 (IFileSystemObject)new FileInfo(fullPath, null);
         }
+
+        public override string[] GetLogicalDrives()
+        {
+            return DriveInfoInternal.GetLogicalDrives();
```

Comment:
Do we need the unix version of DriveInfoInternal?  Couldn't this just use:
```C#
return Interop.Sys.GetAllMountPoints().ToArray();
```
and then you can delete that other file?  If it was multiple lines and was going to be used in several places, it would make sense to centralize it, but the wrapper is just one line, and this file already calls into Interop.* stuff all over the place.  Seems like an unnecessary level of indirection.